features/projects/components/project-card/project-card.tsx

```diff
@@ -50,7 +56,9 @@ export function ProjectCard({ project }: ProjectCardProps) {
             <div className={styles.issuesNumber}>{numEvents24h}</div>
           </div>
           <div className={styles.status}>
-            <Badge color={statusColors[status]}>{capitalize(status)}</Badge>
+            <Badge color={statusColors[status]}>
+              {capitalize(statusMappings[status])}
```

Comment:
There's no need for the capitalize function anymore if you simply capitalize the values in the `statusMappings` object. Apart from this the PR looks great. Well done 👍

Reply:
Thanks! Wasn't sure if the call to capitalize was there for any unknown/future reason, will clean that up